integration-test-common/src/test/java/com/datastrato/gravitino/integration/test/container/DorisContainer.java

```diff
@@ -30,6 +30,9 @@ public class DorisContainer extends BaseContainer {
   public static final int FE_HTTP_PORT = 8030;
   public static final int FE_MYSQL_PORT = 9030;
 
+  private static final String DORIS_FE_PATH = "/opt/apache-doris-1.2.7.1-bin-arm64/fe/log/fe.log";
```

Comment:
The current implementation has a problem: if the log rolls over, historical logs will be lost. Can we use testcontainers's `copyFileFromContainer` method to copy logs?

Reply:
OK, let me verify the effect